File: src/006/main.c
# include <stdio.h>

typedef unsigned int u32;

const u32 MAX = 101;

int main () {
	u32 square_of_sums = MAX * (MAX - 1) * MAX * (MAX - 1) / 4;
	u32 sum_of_squares = 0;
	
	for (u32 i = 1; i < MAX; i++) {
		sum_of_squares += i * i;
	}
	
	printf ("%u\n", square_of_sums - sum_of_squares);
}

File: src/006/main.rs

const MAX: u32 = 101;

fn main () {
	let square_of_sums: u32 = MAX * (MAX - 1) * MAX * (MAX - 1) / 4;
	let mut sum_of_squares: u32 = 0;
	
	for i in 1..MAX {
		sum_of_squares += i * i;
	}
	
	println! ("{}", square_of_sums - sum_of_squares);
}

